mm2src/mm2_main/src/lp_ordermatch/best_orders.rs

```diff
@@ -51,6 +52,7 @@ pub struct BestOrdersRequestV2 {
     coin: String,
     action: BestOrdersAction,
     request_by: RequestBestOrdersBy,
+    exclude_mine: Option<bool>,
```

Comment:
No need for this to be an option
```rust
    #[serde(default)]
    exclude_mine: bool,
```

Reply:
[done](https://github.com/KomodoPlatform/atomicDEX-API/pull/1849/commits/face1b7ed194f8ecaf5b2367809b113bc1dce25b)